### ISSUE TITLE
View changes easily in approval chains

### ISSUE THREAD

**Issue by brucebailey** [blueriver/MuraCMS#2060]
I may be missing this, but aside from going into a document that needs approval and going into version history and comparing the last two items I don't see a way to see the difference.  It would be nice to have that pop up right on "View Details".

**Comment by seanschroeder** [comment]
Good idea. Which version would you use to compare by default? Just the last one?
